rails-commenting.rb

```diff
@@ -48,15 +48,15 @@ def update
   def destroy
     @post = BlogPost.find(params[:id])
     if @post.destroy
-      # ---8)
+      # delete specified blog and redirect to index
       redirect_to blog_posts_path
     end
   end
 
-  # ---9)
+  # make private to control access to the blog post create/edit/update methods
```

Comment:
The private keyword makes anything below it only available in the scope of this controller.